types/react-autosuggest/index.d.ts

```diff
@@ -12,7 +12,16 @@
 
 import * as React from 'react';
 
-declare class Autosuggest<T = any> extends React.Component<Autosuggest.AutosuggestProps<T>> {}
+declare class Autosuggest<T = any> extends React.Component<
+    Autosuggest.AutosuggestProps<T>,
+    Autosuggest.AutosuggestState<T>
+> {
+    /**
+     * Autosuggest exposes these class properties to the parent component.
+     * They can be accessed through `ref`.
+     */
+    input: HTMLInputElement | null;
```

Comment:
Is `undefined` more accurate? Or will it actually be `null`?

The Typescript convention is to prefer undefined for most things (unless it's literally going to be a `null`)

Reply:
`undefined` is more accurate. Mistake on my part.